### PR TITLE
mirrorz.d.json: more range

### PR DESCRIPTION
**Pull request by ZenithalHourlyRate**
From ipip database, example:
```
{'country_name': '中国', 'region_name': '黑龙江', 'line': '教育网'} 
{'country_name': '中国', 'region_name': '北京', 'line': '科技网'}
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,5 +95,11 @@ Any mirror site participating in **302 backend** should provide this file. Mirro
...
     + Partial capability: One endpoint with `filter: [ "NOSSL", "SSL", "SSL:centos" ]`, namely force SSL for one `cname` called `centos`. If one user requests with `http://mirrors.edu.cn/centos`, this endpoint would not be redirected.
   - Private mirror site may use a private IP but declare a public range. For example, suppose USTC has a private IP range of 10.0.0.0/8, USTC mirror is located at 10.0.0.1:8080, and when one user inside USTC accesses `mirrors.edu.cn`, its IP is NATed into 202.0.0.0/24, then `mirrors.edu.cn` can resolve the request into `ustccampus` endpoint.
+  - Possible values for `range`
+    + COUNTRY: Must start with `COUNTRY`, then a colon, then ISO country code. Example: `COUNTRY:CN` or `COUNTRY:US`. Default to `CN`.
+    + REGION: Must start with `REGION`, then a colon, then province name. Example: `REGION:BJ` (Beijing) or `REGION:SH` (Shanghai). Default to `BJ`.
+    + LINE: Must start with `LINE`, then a colon, then ISP name. Example: `LINE:CERNET` or `LINE:CHINANET`. Default to `CERNET`. All currently supported values are `CERNET`, `CSTNET`, `CHINANET`, `UNICOM` and `CMCC`.
```
Comment (bianyifan) [inline README.md]: I would prefer to rename it to `ISP`, it is more of a formal term.

File: README.md [hunk 2]
```diff
@@ -93,5 +95,11 @@ Any mirror site participating in **302 backend** should provide this file. Mirro
     + Syntar sugar: By default we assume each endpoint has both A and AAAA, hence `resolve: "mirrors.example.com", filter:["V4", "V6"]` is equivalent to `resolve: "mirrors.example.com", filter:[]`. To be more simple, `resolve:"10.10.10.10", filter: [ "V4" ]` can be rewritten into `resolve: "10.10.10.10", filter: []`. Note that `resolve: "10.10.10.10", filter: ["V6"]` is invalid and the `"V6"` filter will be ignored.
     + Partial capability: One endpoint with `filter: [ "NOSSL", "SSL", "SSL:centos" ]`, namely force SSL for one `cname` called `centos`. If one user requests with `http://mirrors.edu.cn/centos`, this endpoint would not be redirected.
   - Private mirror site may use a private IP but declare a public range. For example, suppose USTC has a private IP range of 10.0.0.0/8, USTC mirror is located at 10.0.0.1:8080, and when one user inside USTC accesses `mirrors.edu.cn`, its IP is NATed into 202.0.0.0/24, then `mirrors.edu.cn` can resolve the request into `ustccampus` endpoint.
+  - Possible values for `range`
+    + COUNTRY: Must start with `COUNTRY`, then a colon, then ISO country code. Example: `COUNTRY:CN` or `COUNTRY:US`. Default to `CN`.
```
Comment (bianyifan) [inline README.md]: Let's clarify the country code is defined in [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).

File: README.md [hunk 3]
```diff
@@ -93,5 +95,11 @@ Any mirror site participating in **302 backend** should provide this file. Mirro
...
+    + REGION: Must start with `REGION`, then a colon, then province name. Example: `REGION:BJ` (Beijing) or `REGION:SH` (Shanghai). Default to `BJ`.
```
Comment (bianyifan) [inline README.md]: And in here it should be a [GB/T 2260-2007](https://openstd.samr.gov.cn/bzgk/gb/newGbInfo?hcno=C9C488FD717AFDCD52157F41C3302C6D) table 1 alphabetic code.

Comment (iBug) [reply README.md]: Supplementary image: [image](https://user-images.githubusercontent.com/7273074/229088560-3d8af064-9a0f-48fe-b6cf-e28e699fb10f.png)

**Pull request merged**